packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -392,6 +394,106 @@ QUnit.module('Events', moduleConfig, () => {
             assert.strictEqual(handlerStub.callCount, 1);
             assert.strictEqual(handlerStubAfterUpdate.callCount, 1);
         });
+
+        QUnit.test(`${eventHandler} should have correct argument fields`, function(assert) {
+            assert.expect(4);
+
+            this.reinit({
+                [eventHandler]: ({ component, element, event, handleElement }) => {
+                    const $resizeHandle = this.getResizeHandles();
+
+                    assert.strictEqual(component, this.instance, 'component field is correct');
+                    assert.strictEqual($(element).is(this.$element), true, 'element field is correct');
+                    assert.strictEqual($(event.target).is($resizeHandle), true, 'event field is correct');
+                    assert.strictEqual($(handleElement).is($resizeHandle), true, 'handleElement field is correct');
+                },
+                dataSource: [{ text: 'pane 1' }, { text: 'pane 2' }]
+            });
+
+            const pointer = pointerMock(this.getResizeHandles().eq(0));
+
+            pointer.start().dragStart().drag(0, 50).dragEnd();
+        });
+    });
+
+    QUnit.test('onItemCollapsed should be called on collapse prev button click', function(assert) {
+        const onItemCollapsed = sinon.stub();
+
+        this.reinit({
+            onItemCollapsed,
+            dataSource: [{ text: 'pane 1' }, { text: 'pane 2' }]
+        });
+
+        const $collapsePrevButton = this.$element.find(`.${RESIZE_HANDLE_COLLAPSE_PREV_PANE_CLASS}`);
```

Comment:
Let's create correspond methods to get prev, next elements in module setup. It allows avoid duplication.